api/v1beta3/tortoise_types.go

```diff
@@ -256,7 +256,8 @@ const (
 	// TortoisePhaseBackToNormal means tortoise was in the emergency mode, and now it's coming back to the normal operation.
 	// During TortoisePhaseBackToNormal, the number of replicas of workloads are gradually reduced to the usual value.
 	//  - Emergency → BackToNormal
-	TortoisePhaseBackToNormal TortoisePhase = "BackToNormal"
+	TortoisePhaseBackToNormal  TortoisePhase = "BackToNormal"
+	TortoisePhaseAutoEmergency TortoisePhase = "AutoEmergency"
```

Comment:
Again, I do not agree adding a new phase. It'll increase our maintenance effort.

Reply:
I will look at it again later i didnt see your comment earlier

Reply:
@sanposhiho Changed to use only emergency mode. Could you review the new test cases as well? I am also using `currentMetric.ContainerResource.Current.Value.IsZero()` which is the default value for int32 though I am not sure what is supposed to be returned when metrics are down. I assume its the default value for the type but I'm not too familiar with golang

Reply:
> I am not sure what is supposed to be returned when metrics are down.

I guess it's correct to assume HPA to return zero when something goes wrong happens with the metric.
You have to check HPA controller implementation to make 100% sure.